pkg/controller/dns/config.go

```diff
@@ -63,6 +63,20 @@ func (h *configHandler) Handle(req router.Request, resp router.Response) error {
 		logrus.Infof("Obtained AcornDNS domain: %v", domain)
 	}
 
+	if !strings.EqualFold(*cfg.LetsEncrypt, "disabled") {
+		// Ensure that we have a Let's Encrypt account ready
+		leUser, err := EnsureLEUser(req.Ctx, cfg, req.Client, domain)
+		if err != nil {
+			return err
+		}
+
+		// Generate wildcard certificate for domain
+		_, err = leUser.EnsureWildcardCertificateSecret(req.Ctx, req.Client, *cfg.AcornDNSEndpoint, domain, token)
```

Comment:
said elsewhere, but i think we need to also handle renew in the dns daemon, not just the handler

Reply:
changed my mind. this is good enough. dont need to also do in a daemon